modules/installation-machines-bare-metal.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-machines-bare-metal-{context}"]
+= Creating the cluster
+
+To create the {product-title} cluster, you provision machines by using the Ingition config
+files that you generated with the installation program.
+
+.Prerequisites
+
+* Your machines must have internet access.
+* Create the required infrastructure for the cluster.
+* You obtained the installation program and generated the Ignition config files
+for your cluster.
+
+.Procedure
+
+. Obtain the {op-system-first} images.
+
+. Use the the ignition config files that you generated by using the installation
```

Comment:
@abhinavdahiya, what else do we need to tell users about installing RHCOS using an ignition config?

Reply:
The OS team can help provide detailed information.
cc @ashcrow @imcleod

Reply:
Ian says that https://github.com/coreos/coreos-installer is basically still right. I'll put together a short module about putting RHCOS on the cluster machines.